_posts/2020-12-25-gitbucket-4.35.0.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "GitBucket 4.35.0 released!"
+date: 2020-12-25 00:00:00
+image: /images/gitbucket-4.35.0/color_theme_editor.png
+categories: gitbucket
+---
+
+Today, we are pleased to announce the availability of new version of Scala based Git centric software development platform [GitBucket 4.35.0](https://github.com/gitbucket/gitbucket/releases/tag/4.35.0)!
+
+Since this release includes various new features, improvements and bug fixes, it's really challenging to cover all changes in this document. For now, let's see the major stuff in this release.
```

Comment:
```suggestion
Since this release includes various new features, improvements, and bug fixes, it's really challenging to cover all changes in this document. For now, let's see the major stuff in this release.
```